style.css

```diff
@@ -37,6 +37,37 @@
   animation-duration: 2s;
 }
 
```

Comment:
- [OPTIONAL] It would be better if you fix some styles issues in your portfolio. It will help you to have a great project or using as the main portfolio. 😉 

![button align](https://user-images.githubusercontent.com/79876959/141385056-447a23ce-da59-466d-9fc5-453619e8a5bb.PNG)
![button-align-2](https://user-images.githubusercontent.com/79876959/141385059-aafe5f47-edf3-43cc-8ecf-fee56d95025c.PNG)
![tech](https://user-images.githubusercontent.com/79876959/141385063-4e9836e0-79f3-4e8f-916c-60c1b0f3833d.PNG)
![contact](https://user-images.githubusercontent.com/79876959/141385065-c02ffe5b-42a5-4578-a53b-3577104754cc.PNG)
In the last image, you are using different values of `font-size` property and the button's text is not aligned.